LBH.AdultSocialCare.Api/V1/Infrastructure/Entities/ResidentialCareAdditionalNeeds.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace LBH.AdultSocialCare.Api.V1.Infrastructure.Entities
+{
+    public class ResidentialCareAdditionalNeeds : BaseEntity
+    {
+        /// <summary>
+        /// Gets or sets the Id
+        /// </summary>
+        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public Guid Id { get; set; }
```

Comment:
We can consider having Id name as `<EntityName>Id` eg `ResidentialCareAdditionalNeedId` going forward. Becomes friendly when doing joins

Reply:
Thanks!